src/components/css/wishlist.css

```diff
@@ -45,3 +45,30 @@
 .wishlist-page > .card:hover .remove-wishlist {
 	top: 0;
 }
+
+.removing-wishlist {
+	position: absolute;
```

Comment:
Maybe you can name it removing-item because you are using it to remove item and not the whole wishlist. same suggestion .remove-wishlist as well. Again this is personal suggestion you can keep it as well no issues there

Reply:
Thanks, Bhushan will keep this in mind. Thanks for the suggestion.